tests/core/snapshots/graphql-conformance-002.md_0.snap

```diff
@@ -0,0 +1,39 @@
+---
+source: tests/core/spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": {
```

Comment:
instead of recreating right snapshots here we can mark failed tests as skip with relevant comment. And for that test we can create an issue to fix this and enable test

Reply:
That is a good strategy to know which stuff are not currently implemented